db/migrate/20231010180442_create_mentee_questionnaires.rb

```diff
@@ -0,0 +1,25 @@
+class CreateMenteeQuestionnaires < ActiveRecord::Migration[7.0]
+  def change
+    create_table "mentee_questionnaires", id: :binary, force: :cascade do |t|
+      t.binary "respondent_id", limit: 16, null: false, foreign_key: true
+      t.string :name, null: false
+      t.string :country, null: false
+      t.string :city, null: false
+      t.string :work_url, null: false
+      t.boolean :writing_ruby, null: false
+      t.string :starting_point, null: false
+      t.boolean :underrepresented_group, null: false
```

Comment:
Need a detail field to go with this one?

Reply:
4 lines grouped together so unsure how to read this. 

Assuming its last line, if checked true then show a text field to capture details like this?
```
t.boolean :underrepresented_group, null: false
t.text :underrepresented_group_details
```

I also noticed we already added a `demographic_underrepresented_group` field on User table. So we only need to add a details field here?

https://github.com/goodscary/firstrubyfriend/blob/5b4afa7934d25c6be06d8d7bde24fbef63239ad6/db/schema.rb#L102

Reply:
Hmm. Good point... should this not be here and add a `demographic_underrepresented_group_details` text column on `users`.

Reply:
PR to add this to Users table: https://github.com/goodscary/firstrubyfriend/pull/16